frontend/src/pages/LogsPage/SearchForm/SearchForm.tsx

```diff
@@ -1,29 +1,47 @@
-import { Form, useFormState } from '@highlight-run/ui'
-import React from 'react'
+import { Box, Form, Stack, Text } from '@highlight-run/ui'
+import React, { useState } from 'react'
 
 type Props = {
 	onFormSubmit: (query: string) => void
-	query: string
+	initialQuery: string
+	startDate: string
+	endDate: string
 }
 
-const SearchForm = ({ query, onFormSubmit }: Props) => {
-	const form = useFormState({
```

Comment:
I found that `useFormState` was freezing when the passed in `query` was changing. I don't know if it doesn't like the default value getting constantly updated but I opted to remove it for now and track the state myself.